pkg/agent/plugin/workloadattestor/k8s/k8s.go

```diff
@@ -407,16 +423,30 @@ func (p *Plugin) Configure(_ context.Context, req *configv1.ConfigureRequest) (r
 		return nil, err
 	}
 
+	var sigstoreVerifier sigstore.Verifier
+	if config.Experimental != nil {
+		if config.Experimental.Sigstore != nil {
+			cfg := newConfigFromHCL(config.Experimental.Sigstore, p.log)
+			verifier := sigstore.NewVerifier(cfg)
+			err = verifier.Init(ctx)
+			if err != nil {
+				return nil, status.Errorf(codes.InvalidArgument, "error initializing sigstore verifier: %v", err)
+			}
+			sigstoreVerifier = verifier
+		}
+	}
```

Comment:
since it is configurable for windows or posix, could you verify it is working on windows now?